src/LibraryManager.Vsix/Json/JsonHelpers.cs

```diff
@@ -96,19 +96,21 @@ public static bool TryGetInstallationState(ObjectNode parent, out ILibraryInstal
 
             foreach (MemberNode child in children.OfType<MemberNode>())
             {
+                // note: the Json parser escapes backslashes in the node's Value text,
+                // so we need to unescape those so that they match the value from the manifest.
                 switch (child.UnquotedNameText)
                 {
                     case ManifestConstants.Provider:
-                        state.ProviderId = child.UnquotedValueText;
+                        state.ProviderId = UnescapeSlashes(child.UnquotedValueText);
```

Comment:
>UnescapeSlashes(child.UnquotedValueText) [](start = 43, length = 40)

Have you looked at using GetCanonicalizedText on the value token here instead? I'm pretty sure it should give you the unescaped text we want.

Reply:
Sounds perfect, except that it's not exposed publicly from the JSON parser code.

Reply:
Figure out how to call it, thanks for the pointer.